components/pages/explorer/RepoTable.tsx

```diff
@@ -235,6 +258,10 @@ const RepoTable = (props: PageContentProps) => {
         showFilterInput={false}
         columnDropdownText={'Columns'}
         allowTSVExport={false}
+        customTypes={{
+          variants_file_id: (props: any) => <VariantsFileIdCell {...props} />,
```

Comment:
Didn't do the types here. Not sure if it's worth the effort tbh if we're just trying to get this out the door at this point.